File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/DeleteCashItem.scala
package tools.packet.cashshop.operation

import client.inventory.Item
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class DeleteCashItem(private var _item: Item) extends PacketInput {
  def item: Item = _item

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/DeleteCashItem.java
package tools.packet.cashshop.operation;

import client.inventory.Item;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record DeleteCashItem(Item item) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}